### DISCUSSION TITLE
Add option to send webserver audit logging to stdout

### DISCUSSION THREAD

**Discussion by ewierschke** [apache/airflow#29770] [Ideas]
**Description**

While we've discovered that webserver audit logging (actions taken within the webserver, cli etc) appears to be written to the log table of the database, we have a need to have these logs written to stdout of the container running the webserver. 
https://github.com/apache/airflow/blob/master/airflow/www/decorators.py#L33-L62
https://github.com/apache/airflow/blob/master/airflow/models/log.py

**Use case / motivation**

Was hoping an option could be added to enable the same log table entries to be written to stdout to enable capture and centralization of all logs.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by GuidoTournois** [comment]
As far as I could see, there are two ways you could reach your goal.

1) Configure access log by gunicorn. This can be done by updating access_logfile in the airflow configuration file,  `airflow.cfg`. This however, does probably not give you the freedom you are looking for.

2) A better option would be to add a new logger for the web server in `DEFAULT_LOGGING_CONFIG` (e.g. `airflow.webserver`) and retrieve this logger in the  `action_logging` decorator. That way, Airflow users can use the proposed way for setting up custom logging configurations as described here:
 https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/logging-tasks.html
 
I am happy to work on option 2, let's see if that is something you could use.
